### PR TITLE
[Coverage] Api unit tests

### PR DESCRIPTION
**Pull request by afbora**
## Describe the PR

 `Api` namespace unit tests to increase code coverage and this PR separated from #2780

## Ready?

- [x] Added unit tests for fixed bug/feature
- [x] Passing all unit tests
- [x] Fixed code style issues with CS fixer and `composer fix`
- [x] Added in-code documentation (if needed)

### CODE REVIEW

File: src/Api/Api.php [hunk 1]
```diff
@@ -710,14 +716,17 @@ public function responseForException($e): array
     /**
      * Upload helper method
      *
+     * move_uploaded_file() not working with unit test
+     * Added debug parameter for testing purposes as we did in the Email class
```
Comment (afbora) [inline src/Api/Api.php:720]: Have an opinion on this matter?

File: src/Api/Collection.php [hunk 2]
```diff
@@ -19,12 +19,39 @@
  */
 class Collection
 {
+    /**
+     * @var Api
```
Comment (bastianallgeier) [inline src/Api/Collection.php]: This should probably be \Kirby\Api\Api, right? There are a few more here, which probably need to have absolute namespaces. I.e. \Exception. Etc.

File: src/Api/Api.php (hunk 1 above)
Comment (bastianallgeier) [reply src/Api/Api.php:720]: I like it!

**Comment by afbora** [comment]
@bastianallgeier DocBlocks updated, i think it's ready.

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**